src/storage_versioned/mod.rs

```diff
@@ -413,11 +418,12 @@ mod test {
 
     #[test]
     fn storage_versioned_rollback_tests(){
-        let storage_path = test_dir("storage_versioned_rollback_tests");
-        clear_path(storage_path.as_str()).unwrap();
+        // Test can't be run when there are no versions of a storage
+        if VERSIONS_STORED == 0 { return }
```

Comment:
I would suggest to fail the test in this case, otherwise we will not see that this feature was not tested at all.

Reply:
Fixed